app/src/androidTest/java/org/mozilla/fenix/helpers/HomeActivityTestRule.kt

```diff
@@ -25,15 +25,17 @@ class HomeActivityTestRule(
     private val skipOnboarding: Boolean = false
 ) :
     ActivityTestRule<HomeActivity>(HomeActivity::class.java, initialTouchMode, launchActivity) {
+    var longTapUserPreference = android.view.ViewConfiguration.getLongPressTimeout()
```

Comment:
nit: Maybe this could be `private val longTapUserPreference = getLongPressTimeout()` ?

Reply:
done, thanks